piptools/scripts/sync.py

```diff
@@ -41,7 +41,11 @@
     help="Add additional index URL to search",
     envvar="PIP_EXTRA_INDEX_URL",
 )  # noqa
-@click.option("--trusted-host", multiple=True, help="Add a custom trusted host")
+@click.option(
+    "--trusted-host",
+    multiple=True,
+    help="Mark this host as trusted, even though it does not have valid or any HTTPS."
+)  # noqa
```

Comment:
There is no need for "noqa" marker. Could you remove it?

Reply:
removed